CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Contributing Guidelines
+
+The contributing guidelines can be found on the [contributing guidelines page](https://obiagba-mary.gitbook.io/git-and-github-training/contributors-guide/contributing-guidelines) of the git and Github training gitbook learning resource.
```

Comment:
Neatly done. Thank you. Approved!

Just leaving this note in case of next time: You can make your branch name reflect the problem you are solving. For example, instead of using `dev`, you could have used `fix-issue-69` or `add-guidelines` etc. The choice is yours.

Well done once again, I hope to see more of your contributions feel free to pick another issue to work on 👍🏽

Reply:
Thanks for the feedback! I will make sure to implement your suggestions.